README.md

```diff
@@ -0,0 +1,64 @@
+# Nonlocal Gravity Wave Flux Prediction
+
+The code in this repository implements three classes of deep learning models to predict atmospheric gravity wave momentum fluxes, provided the atmospheric background conditions as input.
+
+## Models description
+The three kinds of models are:
+1. **M1:** A single column ANN which takes single column atmospheric variables as input and predicts the gravity wave momentum fluxes in the single column (pixel-to-pixel regression)
+2. **M2:** A regionally nonlocal CNN+ANN which takes a (2N+1)x(2N+1) horizontal columns of atmospheric variables as input and predicts the fluxes in the central single column (image-to-pixel regression)
+3. **M3:** A global attention UNet model which taked global horizontal maps of variables as input to predict fluxes over the whole horizontal domain (image-to-image regression)
+The three models are schematically depicted in the animation below
+
+
+
+![Schematic for the three models](https://amangupta2.github.io/images/icml_schematic.gif)
+
+
+
+The models are trained on modern reanalysis ERA5 which resolves part of the atmospheric gravity wave spectrum. Since it does not resolve the mesoscale wave spectrum, the repository provides some functionality to retrain parts of the models M1-M3 trained on ERA5 to be retrained on fluxes obtained from a 1 km global IFS model which resolves the whole mesoscale wave spectrum.
+
+
+## Usage
+
+### Training
+The code to train M1 and M2 is contained in the ann_cnn_training directory. The code is split into multiple files but the main is invoked in training.py. The model training can be submitted as a single GPU task using the batch.sh script using the command:
```

Comment:
I generally use backticks (`) to surround programming terms or directories to make them standout. For example:

```suggestion
The code to train M1 and M2 is contained in the `ann_cnn_training` directory. The code is split into multiple files but the main is invoked in `training.py`. The model training can be submitted as a single GPU task using the `batch.sh` script using the command:
```

Would you be able to use backticks in the rest of the `README.md` too?

Reply:
Fixed this as well